src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -14,125 +14,152 @@ import SentryTypes from 'app/sentryTypes';
-            {tct(
-              'The email [actorEmail] is not a member of your organization. [inviteUser:Invite] them or link additional emails in [accountSettings:account settings].',
-              {
-                actorEmail: <strong>{actor.email}</strong>,
-                accountSettings: <Link to="/settings/account/emails/" />,
-                inviteUser: (
-                  <a
-                    onClick={() =>
-                      openInviteMembersModal({
-                        initialData: [
-                          {
-                            emails: new Set([actor.email]),
-                          },
-                        ],
-                        source: 'suggested_assignees',
-                      })
-                    }
-                  />
-                ),
-              }
-            )}
-          </EmailAlert>
-        )}
-      </React.Fragment>
-    }
-    body={
-      <HovercardBody>
-        {commits !== undefined && (
-          <React.Fragment>
-            <div className="divider">
-              <h6>{t('Commits')}</h6>
-            </div>
-            <div>
-              {commits.slice(0, 6).map(({message, dateCreated}, i) => (
-                <ReasonItem key={i}>
-                  <CommitIcon />
-                  <CommitMessage message={message} date={dateCreated} />
-                </ReasonItem>
-              ))}
-            </div>
-          </React.Fragment>
-        )}
-        {rules !== undefined && (
+class SuggestedOwnerHovercard extends React.Component {
+  static propTypes = {
+    /**
+     * The suggested actor.
+     */
+    actor: PropTypes.oneOfType([
+      // eslint-disable-next-line react/no-typos
+      SentryTypes.User,
+      // Sentry user which has *not* been expanded
+      PropTypes.shape({
+        email: PropTypes.string.isRequired,
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+      }),
+      // Unidentifier user (from commits)
+      PropTypes.shape({
+        email: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+      }),
+      // Sentry team
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+      }),
+    ]),
+
+    /**
+     * The list of commits the actor is suggested for. May be left blank if the
+     * actor is not suggested for commits.
+     */
+    commits: PropTypes.arrayOf(
+      PropTypes.shape({
+        message: PropTypes.string.isRequired,
+        dateCreated: PropTypes.string.isRequired,
+      })
+    ),
+
+    /**
+     * The list of ownership rules the actor is suggested for. Maybe left blank
+     * if the actor is not suggested based on ownership rules.
+     */
+    rules: PropTypes.arrayOf(PropTypes.array),
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      commitsExpanded: false,
+      rulesExpanded: false,
+    };
+  }
```

Comment:
Instead of `constructor`, let's define/initialize `state` as a class property

Reply:
Will do

Reply:
Still needs this